src/balancerd/src/lib.rs

```diff
@@ -568,6 +571,12 @@ impl PgwireBalancer {
             metrics.tenant_pgwire_sni_count(tenant, has_sni).inc();
         }
 
+        if LOG_PGWIRE_CONNECTION_STATUS.get(&configs) {
+            info!(%conn.uuid, "starting new pgwire connection in balancer");
```

Comment:
```suggestion
            debug!(conn_uuid = %conn.uuid, "starting new pgwire connection in balancer");
```

If we use a defined field like `conn_uuid` in all trace logs, we can specifically filter for that in the tracing filter.